src/components/layout/Group/Group.jsx

```diff
@@ -40,7 +40,7 @@ class Group extends React.Component<Props> {
           } else if (this.props.layout[index]) {
             styleObj.flexBasis = this.props.layout[index];
           } else {
-            styleObj.flexBasis = `${100 / this.props.children.length}%`;
+            styleObj.flexBasis = `${100 / React.Children.count(this.props.children)}%`;
```

Comment:
interesting, why this change?

Reply:
That's actually more correct.

Reply:
This is a flow fix

As React$Node is not an array